css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-avatar-dimensions: 16 24 32 36 48 !default;
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-default-size: 32px !default;
+$persona-font-size: $font-size-7 !default;
+$persona-gap-size: $layout-1 !default;
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+$persona-dimension-font-size-ratio: 0.45;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+	font-size: $persona-font-size;
+
+	.persona-avatar {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: $persona-default-size;
+		height: $persona-default-size;
+		border-radius: $border-radius-rounded;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		font-size: $persona-default-size * $persona-dimension-font-size-ratio;
+		overflow: hidden;
+
+		img,
+		svg {
+			display: block;
+			width: 100%;
+			height: auto;
+			max-height: 100%;
+		}
+
+		span {
+			line-height: 1;
+			text-transform: uppercase;
+		}
+	}
+
+	.persona-description :not(:first-child) {
```

Comment:
Does `persona-description` need more subcomponents? I think we probably want to ensure we have defaults to avoid this situation I see in the document: 

<img width="375" alt="image" src="https://github.com/microsoft/atlas-design/assets/30843002/11a4afbf-877c-4137-bf49-70657e69a064">

Reply:
Not sure what you mean here. Can you clarify please?

Reply:
Does it need `persona-description-name`? (I don't love the name) 

Does we really want bare elements in there? I'd think we should use classes to affect the anchor and paragraph.

Reply:
Updated! Actually used the `persona-name` class, and renamed `persona-description` to `persona-details`, for whatever reason today this sounded much better than description.